tools/testsys/src/run.rs

```diff
@@ -101,16 +83,65 @@ pub(crate) struct Run {
     migration_target_version: Option<String>,
 }
 
+/// This is a CLI parsable version of `testsys_config::GenericVariantConfig`.
+#[derive(Debug, Parser)]
+struct CliConfig {
+    /// The repo containing images necessary for conformance testing. It may be omitted to use the
+    /// default conformance image registry.
+    #[clap(long, env = "TESTSYS_CONFORMANCE_REGISTRY")]
+    conformance_registry: Option<String>,
+
+    /// The name of the cluster for resource agents (EKS resource agent, ECS resource agent). Note:
+    /// This is not the name of the `testsys cluster` this is the name of the cluster that tests
+    /// should be run on. If no cluster name is provided, the bottlerocket cluster
+    /// naming convention `{{arch}}-{{variant}}` will be used.
+    #[clap(long, env = "TESTSYS_TARGET_CLUSTER_NAME")]
+    target_cluster_name: Option<String>,
+
+    /// The image that should be used for conformance testing. It may be omitted to use the default
+    /// testing image.
+    #[clap(long, env = "TESTSYS_CONFORMANCE_IMAGE")]
+    conformance_image: Option<String>,
+
+    /// The role that should be assumed by the agents
+    #[clap(long, env = "TESTSYS_ASSUME_ROLE")]
+    assume_role: Option<String>,
+
+    /// Specify the instance type that should be used. This is only applicable for aws-* variants.
+    /// It can be omitted for non-aws variants and can be omitted to use default instance types.
+    #[clap(long, env = "TESTSYS_INSTANCE_TYPE")]
+    instance_type: Option<String>,
```

Comment:
This needs to be updated once https://github.com/bottlerocket-os/bottlerocket-test-system/pull/602 merges I think

Reply:
It does not. We pin to a released version of testsys.